netbout-web/src/main/java/com/netbout/rest/bout/TkInvite.java

```diff
@@ -46,10 +57,23 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 2.14
+ * @checkstyle ClassDataAbstractionCouplingCheck (210 lines)
  */
 final class TkInvite implements Take {
 
     /**
+     * Size of buffer.
+     */
+    private static final int BUFFER_SIZE = 8192;
+
+    /**
+     * RegEx to validate mail.
+     */
+    private static final Pattern MAIL_MASK = Pattern.compile(
+        "[a-z_\\.\\-A-Z0-9]+[@][a-z_\\.\\-A-Z0-9]+"
```

Comment:
@antonini it would be better to use the first pattern from http://www.regular-expressions.info/email.html, without lower case letters, you can uppercase the message before mathing